build_docs

```diff
@@ -628,10 +630,16 @@ if __name__ == '__main__':
             cmd.extend(argv[3:])
             returncode = subprocess.call(cmd)
             exit(returncode)
+        if '--docker-build' == argv[1]:
+            if len(argv) != 3:
+                raise ArgError(
+                    'Usage: build_docs --build-image <image>')
+            image = argv[2]
+            build_docker_image(image)
+            exit(0)
```

Comment:
I do still wonder if something like [`click`](https://click.palletsprojects.com/en/7.x/) would tighten up a lot of this logic for you.